keel-api/src/main/kotlin/com/netflix/spinnaker/keel/rest/ArtifactController.kt

```diff
@@ -39,13 +39,13 @@ class ArtifactController(
   @ResponseStatus(ACCEPTED)
   fun submitArtifact(@RequestBody echoArtifactEvent: EchoArtifactEvent) {
     log.debug(
-      "Received artifact events {} for {}",
```

Comment:
I R confused. Why is this line gone?

Reply:
There's another almost identical statement that provides the actual useful detail

Reply:
But this looks like just the message, not the entire log statement

Reply:
Oops :( fail. I'll fix!